Remotion/Core/Core/Utilities/StringUtility.cs

```diff
@@ -275,9 +275,12 @@ public static string[] ListToStringArray (IList list)
 
     public static string ConcatWithSeparator (IList list, string separator)
     {
+#pragma warning disable 618
       return ConcatWithSeparator (list, separator, null, null);
+#pragma warning restore 618
     }
 
+    [Obsolete ("Use ConcatWithSeperator (IList, string) instead. Parameter 'format' is no longer used.")]

```

Comment:
The version introducing the obsolete attribute needs to be added after the message.

```suggestion
    [Obsolete ("Use ConcatWithSeperator (IList, string) instead. Parameter 'format' is no longer used. (Version 1.21.8)")]
```